### PR TITLE
0.2.0 justify details while creating task

### PR DESCRIPTION
**Pull request by elliotmessi**
repeat data strategy form item;

### CODE REVIEW

File: src/locales/modules/task.ts [hunk 1]
```diff
@@ -98,6 +98,10 @@ const task = {
   "task.train.form.hyperparam.label": { cn: "超参数", en: "Hyper Params", },
   "task.train.form.traintypes.detect": { cn: "目标检测", en: "Object Detection", },
   "task.train.total.label": { cn: "共 {total} 个", en: "Total {total} assets", },
+  "task.train.form.repeatdata.label": { cn: "当数据重复时：", en: "When data repeated: ", },
```
Comment (phoenix-xhuang) [inline src/locales/modules/task.ts:101]: Found duplicate data

File: src/locales/modules/task.ts [hunk 2]
```diff
@@ -106,6 +110,10 @@ const task = {
   "task.mining.form.strategy.label": { cn: "筛选策略", en: "Filter Strategy", },
   "task.mining.form.topk.label": { cn: "得分TOPK", en: "Scount TOP K", },
   "task.label.form.type.newer": { cn: "未标注部分", en: "Unlabel", },
+  "task.mining.form.label.label": { cn: "是否产生新标注", en: "With Annotations", },
+  "task.mining.form.label.no": { cn: "否", en: "No", },
+  "task.mining.form.label.yes": { cn: "是", en: "Yes", },
+  "task.validator.same.param": { cn: "参数key重复", en: "Same key of param", },
```
Comment (phoenix-xhuang) [inline src/locales/modules/task.ts:116]: Duplicate key of params

Comment (elliotmessi) [reply src/locales/modules/task.ts]: fixed

File: src/locales/modules/task.ts (hunk 1 above)
Comment (elliotmessi) [reply src/locales/modules/task.ts]: fixed

**Review by phoenix-xhuang — APPROVED** [review]

**Review by IJtLJZ8Rm4Yr — APPROVED** [review]

**Pull request merged**